tests/test_pyth_account.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+
+from _pytest.logging import LogCaptureFixture
+
+from pythclient.solana import SolanaClient, SolanaPublicKey
+from pythclient.pythaccounts import PythAccount, PythAccountType, PythProductAccount
+
+
+BCH_PRODUCT_ACCOUNT_KEY_DEVNET = '89GseEmvNkzAMMEXcW9oTYzqRPXTsJ3BmNerXmgA1osV'
+BCH_PRICE_ACCOUNT_KEY_DEVNET = '4EQrNZYk5KR1RnjyzbaaRbHsv8VqZWzSUtvx58wLsZbj'
+
+PRODUCT_ACCOUNT_B64_DATA = ('1MOyoQIAAAACAAAAlwAAADAClHlZh5cpDjY4oXEsKb3iNn0OynlPd4sltaRy8ZLeBnN5bWJvbAdCQ0gv'
+                            'VVNECmFzc2V0X3R5cGUGQ3J5cHRvDnF1b3RlX2N1cnJlbmN5A1VTRAtkZXNjcmlwdGlvbgdCQ0gvVVNE'
+                            'DmdlbmVyaWNfc3ltYm9sBkJDSFVTRARiYXNlA0JDSA==')
+
+
+@pytest.fixture
+def solana_pubkey():
+    return SolanaPublicKey("AHtgzX45WTKfkPG53L6WYhGEXwQkN1BVknET3sVsLL8J")
+
+
+@pytest.fixture
+def pyth_account(solana_pubkey, solana_client):
+    return PythAccount(
+        key=solana_pubkey,
+        solana=solana_client,
+    )
+
+
+@ pytest.fixture
+def product_account(solana_client: SolanaClient) -> PythProductAccount:
+    product_account = PythProductAccount(
+        key=SolanaPublicKey(BCH_PRODUCT_ACCOUNT_KEY_DEVNET),
+        solana=solana_client,
+    )
+    product_account.slot = 96866599
+    product_account.attrs = {
+        'asset_type': 'Crypto',
+        'symbol': 'BCH/USD',
+        'quote_currency': 'USD',
+        'description': 'BCH/USD',
+        'generic_symbol': 'BCHUSD',
+        'base': 'BCH'
+    }
+    product_account.first_price_account_key = SolanaPublicKey(
+        BCH_PRICE_ACCOUNT_KEY_DEVNET,
```

Comment:
the reason why I named it devnet is to help myself (or any other user of the library) to understand that the key is for devnet since different network have different keys -- could remove it if you feel it's not necessary

Reply:
No I mean that this isn't necessarily a devnet key. There is overlap that you're using devnet binary data so it is convenient to use devnet keys. However, this is the data in the tests, not devnet. That's what I'm saying.

You could change the key to be something else and the tests would run, but it wouldn't be devnet.

Reply:
got it, will remove!